packages/@webex/plugin-meetings/src/media/index.ts

```diff
@@ -167,7 +170,14 @@ Media.createMediaConnection = (
       config.bundlePolicy = bundlePolicy;
     }
 
-    return new MultistreamRoapMediaConnection(config, debugId);
+    const metrics = new RtcMetrics(webex, debugId);
+
+    return new MultistreamRoapMediaConnection(
+      config,
+      debugId,
+      (data) => metrics.addMetrics(data),
```

Comment:
so now we have the WCME doing getStats() every 5s and giving us result through this callback, but we also still have the SDK StatsAnalyzer calling getStats() every 5s to do its own analysis (to show logs when we don't send/receive packets etc and to send MQE). We should try to optimize it and avoid the duplicate calls to getStats(). I guess we should change the StatsAnalyzer to not call getStats() and instead use the stats results we get through this callback, do you agree?

BTW, this doesn't have to be fixed in this PR, but for sure needs to be addressed as part of this feature.

Reply:
No because the SDK StatsAnalyzer calls transceiver-specific stats which are formatted differently than these metrics getStats calls which are originally called on the PeerConnection. The stats in this PR are also formatted quite differently than the a regular `StatsReport` item, they are compressed and organized into a unified telemetry-friendly format.

Reply:
ok, I'm sure we could refactor StatsAnalyzer and/or the rtcstats code so that we could have StatsAnalyzer working with this data and avoid doing 2x more calls to getStats() than necessary, but it's a bigger piece of work, and not sure if it's really necessary, so for now it's OK to keep it as it is and we'll see if calling getStats() so often actually causes any issues. Also when we have these dumps available to download, the StatsAnalyzer will become largely obsolete, so we could maybe just change it to do getStats() calls once per minute only so that it can still send MQEs. All of that for is something to look into later on, so for now I've created a separate jira for this: https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-445740

Reply:
AFAIK MQEs from web are ignored, so we could even remove it entirely...
I think the plan is for the telemetry service to compute the MQE format and send it over, so eventually we can just keep this rtcstats code.